test/e2e/external_gateways.go

```diff
@@ -59,7 +69,7 @@ var _ = ginkgo.Describe("e2e non-vxlan external gateway through a gateway pod",
 		srcPingPodName   string = "e2e-exgw-src-ping-pod"
 		gatewayPodName1  string = "e2e-gateway-pod1"
 		gatewayPodName2  string = "e2e-gateway-pod2"
-		externalTCPPort         = 80
+		externalTCPPort         = 91
```

Comment:
to avoid clashing with someone else that is already using port 80 (when the container is host-networked)
for these tests the port here doesn't really matter, as long as nobody uses it